contract_interface.md

```diff
@@ -0,0 +1,30 @@
+# eWASM Contract Interface (ECI) Specification; Version 0
+
+The eWASM Contract Interface (ECI) specifies the structure of a contract module.
+
+### Wire format
+
+Every contract must be stored in the [WebAssembly Binary Encoding](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md) format (in short, WASM bytecode).
+
+### Imports
+
+A contract can only import symbols specified in the [Ethereum Environment Interface](./eth_interface.md).
+
+### Exports
+
+A contract must have exactly two exported symbols:
+- `memory`: the shared memory space available for the EEI to write into.
+- `main`: a function with no parameters and no result value.
+
+### Entry point
+
+The method exported as `main` will be executed by the VM.
```

Comment:
The `main` is used in order to have access to the instance memory.

`start` also depends on language support.